Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -283,3 +294,18 @@ const std::string GameListItem::GetWiiFSPath() const
 	return ret;
 }
 
+bool GameListItem::IsElfOrDol() const
+{
+	const std::string name = GetName();
+	const size_t pos = name.rfind('.');
+
+	if (pos != std::string::npos)
+	{
+		std::string ext = name.substr(pos);
+		std::transform(ext.begin(), ext.end(), ext.begin(), ::tolower);
+
+		return ext == ".elf" ||
+		       ext == ".dol";
```

Comment:
The second tab here is wrong. I suggest just putting this on one line, it's very short.